examples/pytorch/graphsage/train_full.py

```diff
@@ -167,5 +83,42 @@ def main(args):
                         help="Aggregator type: mean/gcn/pool/lstm")
     args = parser.parse_args()
     print(args)
+    
+    # load and preprocess dataset
+    data = load_data(args)
+    g = data[0]
+    features = g.ndata['feat']
+    labels = g.ndata['label']
+    train_mask = g.ndata['train_mask']
+    val_mask = g.ndata['val_mask']
+    test_mask = g.ndata['test_mask']
+    in_feats = features.shape[1]
+    n_classes = data.num_classes
+    n_edges = g.number_of_edges()
+    print("""----Data statistics------'
+      #Edges %d
+      #Classes %d
+      #Train samples %d
+      #Val samples %d
+      #Test samples %d""" %
+          (n_edges, n_classes,
+           train_mask.int().sum().item(),
+           val_mask.int().sum().item(),
+           test_mask.int().sum().item()))
```

Comment:
I'm not sure if it is necessary to log dataset statistics.

Reply:
I think it might be better to let users know/be more clear on the scale of the case they are running ahead of the time.
 
But, yeah,  since dgl intrinsic dataset like Cora do auto-logging. I will remove this manual effort for train_full.py (which uses intrinsic dataset unlike node_classification.py)